src/test/java/io/stargate/sgv2/jsonapi/service/cqldriver/CqlSessionCacheFactoryTests.java

```diff
@@ -0,0 +1,46 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.datastax.oss.driver.api.core.metadata.schema.SchemaChangeListener;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import io.stargate.sgv2.jsonapi.config.DatabaseType;
+import io.stargate.sgv2.jsonapi.config.OperationsConfig;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.SchemaCache;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+/** Tests for {@link CqlSessionCacheFactory}. */
+public class CqlSessionCacheFactoryTests {
+
+  @Test
+  public void testSingleton() {
+    // Not a lot to test, just checking it always returns the same instance.
```

Comment:
Not sure this test provides lots of value considering all mocking going on; looks like it's mostly scaffolding.